### PR TITLE
Issues template branch

### PR DESCRIPTION
**Pull request by rajdeepbuilds**
Created the bug issue report template

### CODE REVIEW

**Comment by tamalCodes** [comment]
Hi @rajdeepbuilds , please also add it for Features, Docs, and Others.
You can take reference from [Milan/.github/ISSUE_TEMPLATE](https://github.com/MilanCommunity/Milan/tree/main/.github/ISSUE_TEMPLATE) for the templates but do not just copy paste.

**Comment by rajdeepbuilds** [comment]
@tamalCodes kindy help me da
Vercel- Authorization required to deploy 
This is the error i am getting.
Else i have updated the bugs,features,docs and others issue templates.

**Comment by rajdeepbuilds** [comment]
also i have accidently named the docs template as browser template.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+name: 🐞 Bug
+description: Found a bug? Use this template
+title: "Bug: Write a small description here"
+labels:
+  [
+    "status: todo ⏳",
+    "🐞 bug",
+    "hacktoberfest"
+  ]
```
Comment (tamalCodes) [inline .github/ISSUE_TEMPLATE/bug-report.yml]: I previously said to not just copy paste. We donot have a label called `status: todo ⏳` or `🐞 bug`
Please fix these kind of things across the whole code. Use the current labels that we have, for example `bug`

**Review by tamalCodes — CHANGES_REQUESTED** [review]

**Comment by tamalCodes** [comment]
You don't need to deploy anything. Just push the changes. 
If there is any error we will take care of that.

**Review by rajdeepbuilds — COMMENTED** [review]
@tamalCodes kindly check it now

**Pull request closed without merging**